### PR TITLE
More fixes

### PR DESCRIPTION
**Pull request by alexdowad**
This is my next batched GH-issue-fixin' PR.

Regarding the new cop requested by @agrimm, which checks for paren-challenged method calls nested inside a paren-privileged method call, what do you think about this:

```ruby
method_with_parens(block_taker { method_without_parens 1, 2 })
```

Is it good, or bad?

### CODE REVIEW

**Comment by bbatsov** [comment]
> Is it good, or bad?

This looks fine to me, as it's the nest method call is not something that would result in a method argument for the outer method.

File: config/default.yml [hunk 1]
```diff
@@ -662,6 +662,12 @@ Style/SingleLineBlockParams:
 Style/SingleLineMethods:
   AllowIfMethodIsEmpty: true
 
+Style/SpecialGlobalVars:
+  EnforcedStyle: use_verbose_names
```
Comment (bbatsov) [inline config/default.yml]: I'd say those are "descriptive" and the others are "cryptic". :-)

Let's rename verbose to descriptive or english. I think verbose carries a different connotation.

**Comment by alexdowad** [comment]
Thanks for all the helpful comments. PR updated.

I will probably push several more fixes to this branch over the coming days, but if and when you want to merge it, please go ahead; I can open another one.

**Comment by bbatsov** [comment]
Apart from the failing build, all changes look good. 

Your `ConditionalAssignment` changes are going to conflict with @rrosenblum's open PRs regarding it. Whoever is ready first for a merge won't have to deal with those. :-)

**Comment by alexdowad** [comment]
Updated PR. Fixed build. Also added one new cop... see the last commit.

**Comment by bbatsov** [comment]
I like the new cop - quite useful.

**Pull request merged**